src/frontends/onnx/frontend/src/utils/common.cpp

```diff
@@ -129,6 +130,31 @@ bool is_failsafe_node(const std::shared_ptr<ov::Node>& node) {
     return rt_info.find(FAILSAFE_NODE) != rt_info.end();
 }
 
+void skip_setting_friendly_names(const std::shared_ptr<ov::Node>& node) {
+    auto& rt_info = node->get_rt_info();
+    rt_info[SKIP_SETTING_FRIENDLY_NAME] = true;
+}
+
+bool friendly_name_already_set(const std::shared_ptr<ov::Node>& node) {
+    const auto& rt_info = node->get_rt_info();
+    return rt_info.find(SKIP_SETTING_FRIENDLY_NAME) != rt_info.end();
+}
+
+void set_friendly_names(const std::shared_ptr<ov::Node> op,
+                        const std::shared_ptr<ov::Node> added_bias,
+                        const onnx_import::Node& onnx_node) {
+    const std::string onnx_name = !onnx_node.get_name().empty() ? onnx_node.get_name() : onnx_node.output(0);
+    if (added_bias == nullptr) {
+        op->set_friendly_name(onnx_name);
+        common::skip_setting_friendly_names(op);
+    } else {
+        op->set_friendly_name(onnx_name + "/WithoutBiases");
+        added_bias->set_friendly_name(onnx_name);
+        common::skip_setting_friendly_names(op);
+        common::skip_setting_friendly_names(added_bias);
+    }
```

Comment:
Would it be considered more readable?
```suggestion
    std::string ov_name = !onnx_node.get_name().empty() ? onnx_node.get_name() : onnx_node.output(0);
    if (added_bias) {
        added_bias->set_friendly_name(ov_name);
        common::skip_setting_friendly_names(added_bias);
        ov_name += "/WithoutBiases";
    }
    op->set_friendly_name(ov_name);
    common::skip_setting_friendly_names(op);```